test/LaunchDarkly.EventSource.Tests/LaunchDarkly.EventSource.Tests.csproj

```diff
@@ -25,7 +25,6 @@
 
 	<ItemGroup>
     <ProjectReference Include="..\..\src\LaunchDarkly.EventSource\LaunchDarkly.EventSource.csproj" />
-    <ProjectReference Include="..\..\..\dotnet-test-helpers\src\LaunchDarkly.TestHelpers\LaunchDarkly.TestHelpers.csproj" />
```

Comment:
I have no idea how this file path reference got in here, or why it did _not_ break CI. It's almost certainly my fault - looks like that's a local path I was using for debugging.